src/admin/content-block/content-block.const.ts

```diff
@@ -114,3 +125,9 @@ export const BUTTON_TYPE_OPTIONS: SelectOption<ButtonTypeOptions>[] = [
 	{ label: i18n.t('admin/content-block/content-block___primair'), value: 'primary' },
 	{ label: i18n.t('admin/content-block/content-block___secundair'), value: 'secondary' },
 ];
+
+export const IMAGE_WIDTH_OPTIONS: SelectOption<ImageWidthOptions>[] = [
+	{ label: i18n.t('paginabreedte'), value: 'full-width' },
+	{ label: i18n.t('groot'), value: '500px' },
+	{ label: i18n.t('middelgroot'), value: '400px' },
```

Comment:
Ook al zal dit nog veranderd worden via copy vind ik dat we deze labels met een hoofdletter moeten laten beginnen zodat de app in zijn originele versie niet messy overkomt.

Reply:
Bijvoorbeeld nu met mijn VideoPlayer.